codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -578,6 +579,57 @@ public String toString() {
         return sb.toString();
     }
 
+    public <T> Optional<T> getValueForField(String fieldName, Class<T> clazz) {
```

Comment:
I'm a little wary of the lowercasing of the member name. I'd personally rather not do it until there's a need and for S3 it looks like there's not.

Reply:
I'm fine either way. Only did it so people wouldn't have to worry about using pascal case but probably not a big issue.

Reply:
Updated.